src/scm/gitlab.js

```diff
@@ -147,35 +144,36 @@ class Gitlab {
       this.followPaginate,
       'gitlab'
     )
-      .then(groups => {
-        this.namespaces = [this.user].concat(groups.map(group => group.name));
-        return this.namespaces;
-      })
-      .catch((err) => {
-        showAlert('Failed to get user info.', LEVEL_ERROR);
-      });
+    .then(groups => {
+      this.namespaces = [this.user].concat(groups.map(group => group.name));
+      this.namesToIds.groups = groups.reduce((obj, item) => (obj[item.name] = item.id, obj), {});
+      return this.namespaces;
+    })
+    .catch((err) => {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/c0b889de942411e3a53c4f3be8d2b855a8d0621a/src/scm/gitlab.js#L152"></span>‘err’ is defined but never used.</td>
    </tr>
  </tbody>
</table>